tune_the_model/cli.py

```
@@ -148,9 +152,13 @@ def delete(self) -> str:
 
     @inited
     def wait_for_uploading_finish(self, sleep_for: int = 1):
+        logger.info("We are taking your files on upload")
         while self.status is not TuneTheModelFileStatus.READY:
+            if TuneTheModelFileStatus.FAILED:
+                logger.error("Something went wrong during upload. Please contact us about this issue")

```

I think we should raise an exception here.